starknet-foundry/crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -90,6 +88,10 @@ impl ResourceTracker for CairoHintProcessor<'_> {
     }
 }
 
+trait ForgeHintProcessor {
+    fn start_roll(&mut self, inputs: &[Felt252]) -> Result<(), EnhancedHintError>;
```

Comment:
I think it would be better for this to already have properly typed arguments and handle the conversions/extracting values before the call to `start_roll`.

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/181/commits/742c1a1bd03e30f9a1853e5bab6c4bdb9785db02